dataeng/jobs/analytics/EventExportIncremental.groovy

```diff
@@ -14,6 +14,9 @@ class EventExportIncremental {
     public static def job = { dslFactory, allVars ->
         allVars.get('ENVIRONMENTS').each { environment, env_config ->
             dslFactory.job("event-export-incremental-$environment") {
+                description('It updates S3 path of exporter config and gpg-keys.' +
```

Comment:
Job which groups tracking events by org and exports them to S3 for research purposes.